music21/figuredBass/notation.py

```diff
@@ -371,6 +402,15 @@ def _getFigures(self):
 
         self.figures = figures
 
+        figuresFromNotaCol = []
+
+        for i in range(len(self.origNumbers)):
```

Comment:
Better to iterate over the actual list like this:

```suggestion
        for i, number in enumerate(self.origNumbers):
```

Then you can remove:

`number = self.orgNumbers[i]`

Reply:
Done. Thanks for the optimazation.